sherpa-onnx/python/sherpa_onnx/__init__.py

```diff
@@ -1,5 +1,78 @@
+from typing import Dict, List, Optional
+
 from _sherpa_onnx import Display
 
 from .online_recognizer import OnlineRecognizer
 from .online_recognizer import OnlineStream
 from .offline_recognizer import OfflineRecognizer
+
+
+def encode_contexts(
```

Comment:
Could you move this funciton into a separate file and import it in `__init__.py`?

I suggest we keep the `__init__.py` file as simple as possible.